docs/nextstrain-internal/pathogen-repo-ci.md

```diff
@@ -0,0 +1,54 @@
+# Pathogen Repo CI
+
+Setting up a CI GitHub Action workflow that automatically runs is extremely
+helpful for catching any changes that break pathogen workflows.
+
+The CI GH Action workflow is _not_ intended to check the validity of the results,
+it just ensures that the pathogen workflow still runs to completion.
+
+## Using `pathogen-repo-ci`
+
+Nextstrain maintains a reusable [pathogen-repo-ci][] GitHub Action workflow that
+can be used to create the CI workflow with minimum set up:
+
+```yaml
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  ci:
+    uses: nextstrain/.github/.github/workflows/pathogen-repo-ci.yaml@master
+```
+
+### `pathogen-repo-ci` Requirements
+
+> [!IMPORTANT]
+> If your pathogen repo does not meet these requirements, please update the
+> repo to conform to them to maintain uniformity of Nextstrain pathogen repos.
+
+1. The pathogen repo must have a top level `nextstrain-pathogen.yaml` file.
+2. To run the CI step for a workflow (ingest, nextclade, or phylogenetic), the
+directory must contain a `build-configs/ci/config.yaml` config file.
+3. At least one of the CI steps must be run or else the CI workflow will exit with an error.
```

Comment:
What are "steps" in this context? Presumably not strictly the action steps of the 'nextstrain-build' as (e.g.) the `Log in to docker.io` step will always run, yeah?